python/ray/serve/api.py

```diff
@@ -290,6 +291,8 @@ class ASGIAppWrapper(cls):
             async def __init__(self, *args, **kwargs):
                 super().__init__(*args, **kwargs)
 
+                install_serve_encoders_to_fastapi()
```

Comment:
makes me sad that this is in `api.py` -- could you move the asgiwrapper to its own (or another) file as part of this change or else a follow up?